examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
from looking at the code it doesn't seem like `static-html` is React specific, or I'm missing something?